src/main/java/filter/Parser.java

```diff
@@ -173,13 +174,25 @@ private FilterExpression parseQualifier(Token token) {
         qualifierName = qualifierName.substring(0, qualifierName.length() - 1).trim();
 
         QualifierType type = QualifierType.parse(qualifierName).orElse(QualifierType.FALSE);
-        return parseQualifierContent(type, false);
+        if (type == QualifierType.SORT) {
+            //special case for sort
```

Comment:
Yes, I'm referring to that issue.

Another issue, since I don't think we'll remove that edge case now.